CONTRIBUTING.md

```diff
@@ -22,6 +22,75 @@ You can contribute to KubeVela in several ways. Here are some examples:
 
 For more ways to contribute, check out the [Open Source Guides](https://opensource.guide/how-to-contribute/).
 
+### Commit Conventions
+
+KubeVela follows the [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) and [commit messages best practices](https://chris.beams.io/posts/git-commit/) to improve better history information.
+
+The commit message should be structured as follows:
+
+```
+<type>[optional scope]: <subject>
+
+[optional body]
+```
+
```

Comment:
It would be better the put **examples** under here.

Reply:
yes, please add some examples